src/nets/e2e_asr.py

```diff
@@ -36,15 +36,7 @@ def _subsamplex(x, n):
     return x, ilens
 
 
-# get output dim for latter BLSTM
-def _get_vgg2l_odim(idim, in_channel=3, out_channel=128):
-    idim = idim / in_channel
-    idim = np.ceil(np.array(idim, dtype=np.float32) / 2)  # 1st max pooling
-    idim = np.ceil(np.array(idim, dtype=np.float32) / 2)  # 2nd max pooling
-    idim = np.array(idim, dtype=np.int32)
-    return idim * out_channel  # numer of channels
-
-
+# TODO(kan-bayashi): no need to use linear tensor
```

Comment:
Can you replace linear_tensor() with the other function, and remove this function?
See https://github.com/espnet/espnet/issues/199

Reply:
I will do.

Reply:
`n_batch_axes` option in `L.Linear` is not merged in v 4.3.1.
I will keep it.